source/features/parse-backticks.tsx

```diff
@@ -1,41 +1,22 @@
 import './parse-backticks.css';
 import onetime from 'onetime';
 import {observe} from 'selector-observer';
-import zipTextNodes from 'zip-text-nodes';
 
 import features from '.';
 import {parseBackticks} from '../github-helpers/dom-formatters';
-import parseBackticksCore from '../github-helpers/parse-backticks';
 
 function init(): void {
 	const selectors = [
 		'.BorderGrid--spacious .f4.mt-3', // `isRepoHome` repository description
-		'.js-commits-list-item .mb-1', // `isCommitList` commit message
-		'.js-commits-list-item pre', // `isCommitList` commit description
-		'.Details[data-issue-and-pr-hovercards-enabled] .d-none a.link-gray-dark', // `isRepoRoot` commit message (GHE #4021)
-		'.Details[data-issue-and-pr-hovercards-enabled] .d-none a.Link--primary', // `isRepoRoot` commit message
-		'.commit-title', // `isCommit` commit message
-		'.commit-desc', // `isCommit` commit description
-		'.js-commit .pr-1 > code', // `isPRConversation` pushed commits
-		'.js-details-container .pr-1 > code', // `isCompare` pushed commits
-		'.Box-row .mb-1 a', // `isCompare` open Pull Request title
-		'.blame-commit-message', // `isBlame` commit message
-		'[id^=ref-issue-]', // `isIssue` issue and PR references
-		'[id^=ref-pullrequest-]', // `isPRConversation` issue and PR references
-		'[aria-label="Link issues"] a', // `isIssue`, `isPRConversation` linked issue and PR
-		'.Box-header.Details .link-gray', // `isSingleFile` commit message (GHE #4021)
-		'.Box-header.Details .Link--secondary', // `isSingleFile` commit message
-		'.Box-header.Details pre', // `isSingleFile` commit description
 		'.release-header', // `isReleasesOrTags` Headers
-		'.existing-pull-contents .list-group-item-link', // `isCompare` with existing PR
+		'.Box-row .mb-1 a', // `isCompare` with existing PR
```

Comment:
Too broad

Reply:
This is not added by me, I just moved it around, see 27ff206428408c37a77ae1dd716d3729b78b7b57

That said I agree this is a little too board, but if it's working fine this whole time maybe we can keep it.